lib/system/main.js

```diff
@@ -4,19 +4,7 @@
 // configure SystemJS
 await SystemJS.import('system/app/config.js');
 
-// load angular and bedrock early
-await SystemJS.import('angular');
-const bedrock = await SystemJS.import('bedrock-angular');
-
-// TODO: remove requirejs shim
-global.requirejs = {
-  toUrl: x => x
-};
-
-// load all detected modules by default
+// load detected modules and start angular app
 await SystemJS.import('importAll');
 
-// start angular app
-bedrock.start();
```

Comment:
This is weird. I would expect apps that want to customize their main startup script would override `main.js` and they could reuse `importAll` as-is ... but this makes it such that they have to override both.